### PR TITLE
Add project to CircleCI

### PR DESCRIPTION
**Pull request by marcelomorgado**
Refs: https://github.com/tasitlabs/tasit/issues/56
Missing add project to Circle CI platform (I have no access).

### CODE REVIEW

**Comment by pcowgill** [comment]
Thanks!

I just added the project here https://circleci.com/gh/tasitlabs/workflows/tasit

If this roadblock happens again I can change your permissions

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+          paths:
+            - ./node_modules
+          key: dependency-cache-{{ checksum "package.json" }}
+
+      # run tests!
+      - run:
+          name: test
+          command: npm test
+
+      - store_artifacts:
```
Comment (pcowgill) [inline .circleci/config.yml]: FYI this step and the one after it isn't working well for the Tasit SDK repo, so we should remove it here

Check out the steps below `test` here https://circleci.com/gh/tasitlabs/TasitSDK/23

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+      - save_cache:
+          paths:
+            - ./node_modules
+          key: dependency-cache-{{ checksum "package.json" }}
+
+      # run tests!
+      - run:
+          name: test
+          command: npm test
```
Comment (pcowgill) [inline .circleci/config.yml]: I forget, does this do a linting step? If not, we could add a step for that here too. But that could also come in a later PR and we could track it with an issue.

**Comment by pcowgill** [comment]
@marcelomorgado Now that I merged the other branch (and corresponding PR) that this branch was based off of, can you please merge the latest changes from develop into this feature branch when you get a chance? Thanks!

File: .circleci/config.yml (hunk 1 above)
Comment (marcelomorgado) [reply .circleci/config.yml]: Done.

File: .circleci/config.yml (hunk 2 above)
Comment (marcelomorgado) [reply .circleci/config.yml]: Done

**Comment by marcelomorgado** [comment]
> @marcelomorgado Now that I merged the other branch (and corresponding PR) that this branch was based off of, can you please merge the latest changes from develop into this feature branch when you get a chance? Thanks!

Is it OK now?

**Comment by pcowgill** [comment]
> > @marcelomorgado Now that I merged the other branch (and corresponding PR) that this branch was based off of, can you please merge the latest changes from develop into this feature branch when you get a chance? Thanks!
> 
> Is it OK now?

Yep!

**Comment by pcowgill** [comment]
There's one log statement that's at error level, even though the tests are still passing. (But this won't gate merging this PR.)

```
    console.error node_modules/prop-types/checkPropTypes.js:19
      Warning: Failed prop type: The prop `dataSource` is marked as required in `ListLands`, but its value is `undefined`.
          in ListLands (at ListLandsScreen.js:54)
```

I guess we could change that to an optional prop if we want to keep allowing for an undefined one. Alternatively, we can make sure it's always defined. We can create a `good first issue` for that, perhaps. What do you think?

**Comment by marcelomorgado** [comment]
> There's one log statement that's at error level, even though the tests are still passing. (But this won't gate merging this PR.)
> 
> ```
>     console.error node_modules/prop-types/checkPropTypes.js:19
>       Warning: Failed prop type: The prop `dataSource` is marked as required in `ListLands`, but its value is `undefined`.
>           in ListLands (at ListLandsScreen.js:54)
> ```
> I guess we could change that to an optional prop if we want to keep allowing for an undefined one. Alternatively, we can make sure it's always defined. We can create a `good first issue` for that, perhaps. What do you think?

There is a reason why I've kept this warning message, but I don't remember. I've just created this issue: https://github.com/tasitlabs/tasit/issues/60

**Pull request merged**